docs/api-reference/front-commerce-core/create-resized-image-response.mdx

```diff
@@ -0,0 +1,116 @@
+---
+title: "createResizedImageResponse"
+sidebar_position: 2
+description:
+  "The createResizedImageResponse allows to create custom loaders which allow
+  resizing and caching images on the fly."
+---
+
+The `createResizedImageResponse` allows to create custom loaders which allow
+resizing and caching images on the fly.
+
+## Usage
+
+The `createResizedImageResponse` function has 3 parameters:
+
+- `request`:The request to handle.
+- `fetchRemoteImage`: A function that returns the image buffer.
+- `cacheConfig`: The optional
+  [cache configuration](https://gitlab.blackswift.cloud/front-commerce/front-commerce/-/blob/fdcbea5288ca66b12e4c839653bcbe033114a3e9/packages/core/resized-image/createResizedImageResponse.ts#L17-22)
+  which handles the caching of an image
+
+This can be used to serve images either from a remote service, or from a local
+file system, next we will see examples for both cases.
+
+### Remote image
+
+For this example, we will create a
+[splat route](https://remix.run/docs/en/1.19.3/file-conventions/route-files-v2#splat-routes)
+that will fetch an image from a remote service.
+
+To do so, we will introduce the `createResizedImageResponse` function.
+
+```ts title="/app/routes/media.prismic.$.ts"
```

Comment:
```suggestion
```ts title="/app/routes/media.picsum.$.ts"
```

**thought:** we could at some point start a `3.x` directory under the examples repository with these example routes

Reply:
Yea that is the idea.